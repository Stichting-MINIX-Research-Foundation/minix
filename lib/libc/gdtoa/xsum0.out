README	664d441	15750
arithchk.c	ebbe5bc7	4075
dmisc.c	c8daa18	4682
dtoa.c	f6ed7306	17542
g_Qfmt.c	f6aad16c	2926
g__fmt.c	e961947e	3401
g_ddfmt.c	1f75ef12	4161
g_dfmt.c	24be3a8	2629
g_ffmt.c	197652f1	2516
g_xLfmt.c	3fd29c5	2773
g_xfmt.c	f6a580a	2862
gdtoa.c	e4651ba8	17355
gdtoa.h	f966845d	4944
gdtoa_fltrnds.h	1aaf5112	421
gdtoaimp.h	395c83e	20044
gethex.c	144ecbb4	7107
gmisc.c	1859d016	2084
hd_init.c	efdbe921	1797
hexnan.c	891b04e	3468
makefile	ffae86d0	3148
misc.c	1d84f517	14416
printf.c	15c97ab	158
printf.c0	efece124	29696
qnan.c	efd33d64	3417
smisc.c	feb7290	3699
stdio1.h	e6e5dc11	2909
strtoIQ.c	1809dfcf	1939
strtoId.c	f41ddac2	1931
strtoIdd.c	f13e3bc3	2105
strtoIf.c	f12c6af4	1875
strtoIg.c	fe47bc72	3591
strtoIx.c	e50f716d	1960
strtoIxL.c	ea0b821b	1931
strtod.c	1dabf6	22744
strtodI.c	f1dbb0af	4004
strtodg.c	eacc7424	21188
strtodnrp.c	af895e9	2538
strtof.c	e1cb24dd	2161
strtopQ.c	f244b012	2645
strtopd.c	1c3c9ce	1751
strtopdd.c	177251c9	4640
strtopf.c	f305c6ea	2155
strtopx.c	1529e1f5	2710
strtopxL.c	69418e3	2469
strtorQ.c	9360a0b	2885
strtord.c	af5c50e	2491
strtordd.c	e3d20ab	4992
strtorf.c	fbe2ea18	2402
strtorx.c	168faee5	2957
strtorxL.c	42c6cec	2718
sum.c	f525bad9	2494
ulp.c	edb56c0b	1866
