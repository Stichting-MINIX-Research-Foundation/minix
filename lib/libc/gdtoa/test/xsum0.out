README	efba0d5d	3412
Q.ou0	e4592b85	28742
Q.ou1	ea0b344d	39572
Qtest.c	efe2b3f4	5116
d.out	f271efc9	28131
dI.out	d522eef	4369
dIsi.out	1dd6d02f	4350
dItest.c	e33800ce	2371
dd.out	e262456e	40923
ddsi.out	1f94bbe2	10251
ddtest.c	ef71cbf3	4986
dt.c	e562c302	6808
dtest.c	9a5d01	4080
dtst.out	e284ac98	23711
f.out	9013e91	21537
ftest.c	1c824a88	4001
getround.c	ee95ed1	2502
makefile	f714a641	5634
pfLqtestnos	ffb9723	99
pftest.c	ea314d7f	3452
pftestQ.out	198434fd	830
pftestx.out	1ccea5dd	788
pftestLq.out	1691dbfc	845
pftestnos	ecbc9be6	101
rtestnos	f94bcdf6	336
strtoIdSI.c	7bfb88b	49
strtoIddSI.c	72e8852	50
strtodISI.c	ed08b740	49
strtodt.c	f1aa53af	3374
strtopddSI.c	13e7138d	50
strtorddSI.c	f7e4b1d5	50
testnos	e89999d6	485
testnos1	7e16229	294
testnos3	f5ae7ef3	14403
x.ou0	1402f834	25372
x.ou1	f1af5a00	34581
xL.ou0	faa3a741	26363
xL.ou1	e349e5c	37165
xLtest.c	ee11f673	4843
xQtest.c	efbe29be	1912
xtest.c	ec9f5deb	4834
